### PR TITLE
Ryna - L2232.electricity_FERC

### PR DESCRIPTION
**Pull request by rynacui**
#718

### CODE REVIEW

**Comment by rynacui** [comment]
@bpbond I'm going to try the batch chunk `batch_electricity_USA_xml`

File: R/zchunk_L2232.electricity_FERC_USA.R [hunk 1]
```diff
@@ -51,254 +55,489 @@ module_gcam.usa_L2232.electricity_FERC_USA_DISABLED <- function(command, ...) {
...
+        L2232.SubsectorLogit_USAelec
+
+      # L2232.TechShrwt_USAelec: technology share-weights in USA electricity trade
+      A232.structure %>%
+        filter(region == "USA") %>%
+        select(one_of(LEVEL2_DATA_NAMES[["Tech"]])) %>%
+        repeat_add_columns(tibble(grid_region = grid_regions)) %>%
+        repeat_add_columns(tibble(year = MODEL_YEARS)) %>%
+        mutate(subsector = replace(subsector, grepl("grid_region", subsector),
+                                   paste(grid_region[grepl("grid_region", subsector)], "electricity trade", sep = " ")),
```
Comment (bpbond) [inline R/zchunk_L2232.electricity_FERC_USA.R]: " " is default for `paste` so no real need for it to be specified, but doesn't hurt.

File: R/zchunk_batch_electricity_USA_xml.R [hunk 2]
```diff
@@ -10,76 +10,70 @@
 #' original data system was \code{batch_electricity_USA_xml.R} (gcamusa XML).
 module_gcamusa_batch_electricity_USA_xml_DISABLED <- function(command, ...) {
```
Comment (rynacui) [inline R/zchunk_batch_electricity_USA_xml.R]: @bpbond I made several changes for the batch, but I'm can't enter the debug mode. So it is still disabled.

**Comment by bpbond** [comment]
Holding on @gokuliyer review.

**Pull request merged**